data/src/main/java/me/androidbox/data/alarm_manager/AlarmReceiver.kt

```diff
@@ -25,15 +32,26 @@ class AlarmReceiver : BroadcastReceiver() {
             }
         }
     }
-    
+
     private fun createAndShowNotification(context: Context, agendaId: String, title: String, description: String) {
         val notificationManager = getNotificationManager(context)
 
+        /** TODO Only Events for the moment, tasks and reminders coming soon... */
+        val agendaIntent = Intent(
+            Intent.ACTION_VIEW,
+            "$EVENT_DEEPLINK={$agendaId}".toUri())
```

Comment:
Is this correct in passing the agendID to the deeplink? This is only for Events at the moment

Reply:
The event id? Yea its correct